.vscode/extensions.json

```diff
@@ -13,7 +13,9 @@
 			"usernamehw.errorlens",
 			"ms-azuretools.vscode-azurefunctions",
 			"GitHub.vscode-pull-request-github",
-			"Gruntfuggly.todo-tree"
+			"Gruntfuggly.todo-tree",
+			"ms-azuretools.vscode-azurefunctions",
+			"ms-python.python",
```

Comment:
These two rows are twice in there - row 14 (azure functions) and I think Python itself is somewhere further up